### PR TITLE
fix(curriculum): external orbital link in legacy JSA

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

clear task 41 in #40749

Although in preview it looks fine

image

It's broken when I checked it in demo

image

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/map-the-debris.md [hunk 1]
```diff
@@ -8,12 +8,21 @@ dashedName: map-the-debris
 
 # --description--
 
+The orbital period is the amount of time a given astronomical object takes to complete one orbit around another object.
+
+According to Kepler's Third Law, the orbital period T of two point masses orbiting each other in a circular or elliptic orbit is:
+
+${T=2\pi {\sqrt {\frac {a^{3}}{\mu }}}}$
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/map-the-debris.md:15]:
```suggestion
$$
T = 2 \pi \sqrt{\frac{a^{3}}{\mu}}
$$
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
This is just for formatting to look nice.

Otherwise, these changes are also needed: #46270

**Review by ShaunSHamilton — DISMISSED** [review]
LGTM 👍

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Hold off on merging this for a moment - going to play with the formatting a bit

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/map-the-debris.md [hunk 2]
```diff
@@ -8,12 +8,23 @@ dashedName: map-the-debris
...
+According to Kepler's Third Law, the orbital period T of two point masses orbiting each other in a circular or elliptic orbit is:
+
+$$
+T = 2 \pi \sqrt{\frac{a^{3}}{\mu}}
+$$
+
+- a is the orbit's semi-major axis
+- μ = GM is the standard gravitational parameter
+- G is the gravitational constant,
+- M is the mass of the more massive body.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/map-the-debris.md:22]:
```suggestion
According to Kepler's Third Law, the orbital period $T$ of two point masses orbiting each other in a circular or elliptic orbit is:

$$
T = 2 \pi \sqrt{\frac{a^{3}}{\mu}}
$$

- $a$ is the orbit's semi-major axis
- $μ = GM$ is the standard gravitational parameter
- $G$ is the gravitational constant,
- $M$ is the mass of the more massive body.
```
We should wrap the variables in inline mathjax both for formatting and to facilitate translation.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
Still LGTM 👍

**Review by naomi-lgbt — APPROVED** [review]
I'm happy with this now (:

**Pull request merged**